Benutzerdoku/Benutzerdoku.tex

```diff
@@ -0,0 +1,18 @@
+\documentclass{scrartcl}
+
+\usepackage[ngerman]{babel}
+\usepackage{graphicx}
+\usepackage{wrapfig}
+\usepackage[section]{placeins}
+
+\begin{document}
+	\tableofcontents
+	
+	\title{Prüfungsplaner \\ Benutzerdokumentation}
+	\author{Gruppe 7}
+	\date{Sommersemester 2018}
+	\maketitle
+
```

Comment:
Evtl. vorher noch ein kurzer Absatz (einfach direkt hier rein, nicht in ne extra Datei) was das System überhaupt macht?